src/System.Net.Http/src/System/Net/Http/Headers/ProductInfoHeaderValue.cs

```diff
@@ -155,7 +155,7 @@ internal static int GetProductInfoLength(string input, int startIndex, out Produ
             }
             else
             {
-                // Trailing whitespaces are removed by GetProductLength().
+                // Trailing whitespace are removed by GetProductLength().
```

Comment:
I wasn't questioning the change, but rather whether the subsequent word should be "is" rather than "are".